src/main/groovy/com/netflix/gradle/plugins/packaging/ProjectPackagingExtension.groovy

```diff
@@ -35,8 +37,12 @@ public class ProjectPackagingExtension extends SystemPackagingExtension {
     public ProjectPackagingExtension(Project project) {
         FileResolver resolver = ((ProjectInternal) project).getFileResolver();
         Instantiator instantiator = ((ProjectInternal) project).getServices().get(Instantiator.class);
-        delegateCopySpec = new DefaultCopySpec( resolver, instantiator);
-
+        if (GradleVersion.current().baseVersion >= GradleVersion.version("6.0")) {
```

Comment:
Will this work with 6.0-rc or snapshots?

Reply:
Yep, `baseVersion` is doing the trick. 6.0-rc and 6.0-snapshot have both` baseVersion = 6.0`

Reply:
interesting, that's good to know! thanks